manifests/m/Microsoft/AzureAztfy/0.9.0/Microsoft.AzureAztfy.installer.yaml

```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Microsoft.AzureAztfy
+PackageVersion: 0.9.0
+InstallerLocale: en-US
+Scope: machine
+UpgradeBehavior: install
+Commands:
+- aztfy
+Installers:
+- Architecture: x64
+  InstallerType: wix
+  InstallerUrl: https://github.com/Azure/aztfy/releases/download/v0.9.0/aztfy_v0.9.0_x64.msi
+  InstallerSha256: BCDEE5162DB37D8472EF4947F38F1A01C753D6A13C7ED9EE2748B6E63CE38E50
+  ProductCode: '{D8F51C74-F844-4887-8940-5D80AFEB7B63}'
+  AppsAndFeaturesEntries:
+  - DisplayName: aztfy
+    Publisher: Microsoft
+    ProductCode: '{566C44B9-7312-4841-8537-0FB95CDFD2A6}'

```

Comment:
@russellbanks Could you please explain why shall we update the `ProductCode` here?

Reply:
> @russellbanks Could you please explain why shall we update the `ProductCode` here?

The `AppsAndFeaturesEntries` needs to match what is written to the registry. This means that the `ProductCode` under `AppsAndFeaturesEntries` needs to match the Product Code of the application.

The ProductCode wasn't necessary to have in AppsAndFeaturesEntries, I removed it in https://github.com/microsoft/winget-pkgs/pull/88742. This means that future automation should be able to run without changes being requested by moderators.

I have also added the x86 installer, so that should be taken into account, too, in case the automation needs changing for that.

Reply:
@magodo, I've created https://github.com/Azure/aztfy/pull/304 to add the x86 installer to the automation.